common/g2d.c

```diff
@@ -404,6 +404,7 @@ int g2d_polygon_contains_point(const zarray_t *poly, double q[2])
     int v = (quad_acc >= 2) || (quad_acc <= -2);
 
     if (0 && v != g2d_polygon_contains_point_ref(poly, q)) {
+        // disabled - used for debug only
         printf("FAILURE %d %d\n", v, quad_acc);
         exit(-1);
     }
```

Comment:
If this is only for debugging and cannot be controlled at runtime, then this should be guarded by an #ifdef so that this "dead code" does not get compiled and we can skip the if condition.

Better, this should probably be an `assert` so that it gets automatically tested in debug mode but not compiled into release builds.

I know, it's not related to the problem you try to solve but if you are in the mood, you could still add a commit for this :-)